bigbang/archive.py

```diff
@@ -70,9 +82,7 @@ def __init__(self, data, archive_dir=CONFIG.mail_path, mbox=False):
         if isinstance(data, pd.core.frame.DataFrame):
             self.data = data.copy()
         elif isinstance(data, str):
-            self.data = mailman.load_data(
-                data, archive_dir=archive_dir, mbox=mbox
-            )
+            self.data = load_data(data, archive_dir=archive_dir, mbox=mbox)
```

Comment:
The documentation of this __init__ function is incorrect and incomplete. There is no single_file parameter. I don't know what the mbox parameter means. In this case, all the parameters are just being passed on directly to `load_data`, but we should have the documentation clear here from the start.

Reply:
I'm reading the docstring for this and while it has a rather convoluted design, I don't see what you see as incorrect. There's no mention of a 'single_file' parameter. So I'm not sure how to address what you're saying.